_targets.R

```diff
@@ -253,6 +259,19 @@ list(
              format = "file"
   ),
   
+  ##generate table of data to download from sciencebase
+  tar_target(p1_make_sb_dl_table,
+             makedltable(idtostart, outdir = "./1_fetch/out"),
+             format = "file"),
+  
+  ##generate table of landscape data for gages list  
+  tar_target(p1_make_sb_landscapedata,
+             downloadchildren(gages=gagesii, dldir = "./1_fetch/out/dldir", 
```

Comment:
There isn't actually a dataframe called 'gagesii' just the 'gagesii_path'. In there very first target (lines 106-110), there is an internal function that reads the filepath and does some quick manipulation. I would suggest creating a new data frame (named gagesii) using that same function from lines 106-110, placing it after line 92, then replacing the internal function in the first target with the data frame callout. Then we can use the callout here as well.